src/core/ServerPolyfillList.ts

```diff
@@ -49,9 +49,9 @@ const ELECTRON_POLYFILL = new Set<string>(
     ]
 );
 export function isServerPolyfill(name: string) {
-    return SERVER_POLYFILL.has(name);
+    return SERVER_POLYFILL.has(name) || false;
```

Comment:
I think `Set.has`returns a boolean by default

Reply:
Hum yep i am stupid i go revert the commit and replace by real fix